Add more questions to basic Next.js tutorial. (#813)
vercel/next-site: pages/learn/basics/dynamic-routes/page-path-external-data.mdx

}
```

Now, here’s what’s new: We’ll export an async function called [`getStaticPaths`](/docs/basic-features/data-fetching#getstaticpaths-static-generation) from this page.

> In this function, we need to return a list of **possible values** for `id`. .

Reviewer: ```suggestion
In this function, we need to return a list of **possible values** for `id`.
```

}
```

Finally, we need to implement [`getStaticProps`](/docs/basic-features/data-fetching#getstaticprops-static-generation) again - this time, to fetch necessary data for the blog post with a given `id`.

> [`getStaticProps`](/docs/basic-features/data-fetching#getstaticprops-static-generation) is given `params`, which contains `id` (based on variable defined in the file name).

Reviewer: What about:

```suggestion
[`getStaticProps`](/docs/basic-features/data-fetching#getstaticprops-static-generation) is given `params`, which contains `id` (because the file name is `[id].js`).
```
